### ISSUE TITLE
Whether the object position is strictly required to be consistent？

### ISSUE THREAD

**Issue by GlassyWing** [xiahaifeng1995/PaDiM-Anomaly-Detection-Localization-master#1]
This method seems to be very sensitive to the position of the object. Will the displacement of the object in the image have a significant effect?

**Comment by plutoyuxie** [comment]
Yes, indeed. So this point is a direction worthy of improvement.

**Comment by kumar10725** [comment]
@GlassyWing Not only the position of the objects needs to be consistent but also you need to have access to the test data beforehand in order to find optimal threshold. Which being said defeats the purpose, this method is however more accurate but not robust at all.

**Comment by Chavo11** [comment]
@GlassyWing Can I ask you if you're refering to this particular implementation or to the paper itself?

**Comment by bjaeger1** [comment]
the object position is not required to be consistent
